client/src/components/MeetingExperience.tsx

```diff
@@ -64,6 +73,11 @@ export const MeetingExperience = (props: MeetingExperienceProps): JSX.Element =>
           endpointUrl,
           chatEnabled
         );
+        adapter.on('callEnded', async () => {
```

Comment:
Doesn't matter much but probably doesn't need to be `async` since we're not awaiting anything?

Reply:
More importantly, if we don't have any postCall, then maybe we shouldn't subscribe to `callEnded` at all? Instead of always subscribing and then checking if we actually need the survey in the handler.

Reply:
Thanks, have made this change.